src/main/steps/edge-case/address/select/content.test.ts

```diff
@@ -66,7 +69,7 @@ describe('applicant1 > address > select > content', () => {
     const commonContent1 = { language: 'cy', userCase: { applyingWith: 'alone' } } as CommonContent;
```

Comment:
1) do we need applyingWith?

2) Also create constants for 'en' and 'cy'

const EN = 'en';
const CY = 'cy';